mgbdis.py

```diff
@@ -649,17 +655,51 @@ def process_data_in_range(self, rom, start_address, end_address, arguments = Non
                 self.append_output(self.format_data(values))
                 values = []
 
+    def get_character_map_index(self, arguments):
+        #no args
+        if arguments == None:
+            return -1
+        #not charmap arg
+        if not "cm" in arguments and not "charmap" in arguments:
+            return -1
+        name = arguments.split("=")[-1]
+        #no index specified
+        if name == "cm" or name == "charmap":
+            return 0
```

Comment:
`arguments` is a string, right? If so, this seems like a better idea, in case other `=`-using arguments are introduced:
```suggestion
        name = None
        for argument in arguments.split(":"):
            key_value = argument.split("=", 1)
            if len(key_value) == 2:
                name,value = key_value
                if key == "cm" or key == "charmap":
                    break
        else:
            abort(...)
```
(Sorry, my Python is a little rusty, so hopefully the scoping rules work out here.)